src/status_im/signals/core.cljs

```diff
@@ -67,6 +67,8 @@
       "envelope.expired"   (transport.message/update-envelopes-status cofx (:ids (js->clj event-js :keywordize-keys true)) :not-sent)
       "message.delivered"  (let [{:keys [chatID messageID]} (js->clj event-js :keywordize-keys true)]
                              (models.message/update-db-message-status cofx chatID messageID :delivered))
+     ;; "message.seen"       (let [{:keys [chatID messageID]} (js->clj event-js :keywordize-keys true)]
```

Comment:
could you elaborate please?

Reply:
I forgot also to remove this line. I had been working on the develop branch while working on another feature. I sorted that let me remove this line of code